std/ascii.d

```diff
@@ -314,7 +314,7 @@ bool isUpper(dchar c) @safe pure nothrow @nogc
   +/
 bool isDigit(dchar c) @safe pure nothrow @nogc
 {
-    return '0' <= c && c <= '9';
+    return (cast(ubyte)(c - '0')) < 10;
```

Comment:
this would give incorrect results for `cast(dchar)('0'+256)`

Reply:
uh did I forget to change it to uint?
let me fix that